job-dsls/jobs/pr_jobs.groovy

```diff
@@ -50,12 +50,6 @@ def final REPO_CONFIGS = [
         "droolsjbpm-knowledge"      : [
                 label: "kie-rhel7 && kie-mem4g"
         ],
-        "drools"                    : [
-                sonarCloudProjectKey: "org.drools:drools"
-        ],
-        "optaplanner"               : [
-                sonarCloudProjectKey: "org.optaplanner:optaplanner"
-        ],
```

Comment:
Just to double-check, these are leftovers, aren't they?

Reply:
Yes, as I reworked the profile for sonarcloud analysis (see kie-parent), we no longer need the project key here. Either the repository overrides the profile or it will just use `groupId:artifactId`.

Reply:
Actually this would remove the PR check jobs, I meant to remove just the sonarCloudProjectKey properties. Fixed.

Reply:
Ah, true. Thanks for explanation.